nbgrader/exchange/ngshare/list.py

```diff
@@ -65,34 +53,25 @@ def _get_assignments(self, course_ids):
         """
         assignments = []
         for course_id in course_ids:
-            url = '{}{}/assignments/{}'.format(self.ngshare_url, self.prefix,
-                                               course_id)
-            params = {'user': self.username}
-
-            try:
-                response = requests.get(url, params=params)
-                self.check_response(response)
-            except Exception as e:
-                self.log.error('Failed to get assignments from course {}. '
-                               'Reason: {}'.format(course_id, e))
+            response = self.ngshare_api_get('/assignments/{}'.format(course_id))
+            if response is None:
+                self.log.error('Failed to get assignments from course {}.')
```

Comment:
Format string with `course_id`.

Reply:
Fixed

Reply:
Did the change make it into the commit?
```suggestion
                self.log.error('Failed to get assignments from course {}.'.format(course_id))
```

Reply:
Now it has.